skimage/measure/fit.py

```diff
@@ -457,31 +456,28 @@ class EllipseModel(BaseModel):
     where ``(xt, yt)`` is the closest point on the ellipse to ``(x, y)``. Thus
     d is the shortest distance from the point to the ellipse.
 
-    This estimator minimizes the squared distances from all points to the
-    ellipse::
-
-        min{ sum(d_i**2) } = min{ sum((x_i - xt)**2 + (y_i - yt)**2) }
-
-    Thus you have ``2 * N`` equations (x_i, y_i) for ``N + 5`` unknowns (t_i,
-    xc, yc, a, b, theta), which gives you an effective redundancy of ``N - 5``.
+    This estimator minimizes
+    The estimator is based on a least squares minimization. The optimal
+    solution is computed directly, no iterations are required. This leads
+    to a simple, stable and robust fitting method.
 
     The ``params`` attribute contains the parameters in the following order::
 
         xc, yc, a, b, theta
 
-    A minimum number of 5 points is required to solve for the parameters.
-
     Attributes
     ----------
     params : tuple
-        Ellipse model parameters in the following order `xc`, `yc`, `a`,
-        `b`, `theta`.
+        Ellipse model parameters in the following order `xc`, `yc`, `a`, `b`,
+        `theta`.
```

Comment:
It looks like the `params` attribute will be the output of `nan_to_num`, which is a NumPy array, not a tuple?